packages/theming/atlas/src/themesource/atlas_core/web/design-properties.json

```diff
@@ -208,6 +236,12 @@
             "type": "Toggle",
             "description": "Hide element on desktop.",
             "class": "hide-desktop"
+        },
+        {
+            "name": "Width 100",
```

Comment:
This name is very confuse! 100 pixels? 100 percent?

Reply:
Agreed, we can add % behind the 100